exercises/react-vite/03-intro-to-jsx/02-solution/src/App.test.tsx

```diff
@@ -15,7 +15,7 @@ describe('App component', () => {
 
   it('renders the title', () => {
     render(<App />);
-    const titleElement = screen.getByText(/Ordering food has never been easier/i);
+    const titleElement = screen.getByRole('heading', { level: 1, name: /Ordering food has never been easier/i })
```

Comment:
It might be too early to bring this in, but I like letting people know that RTL can be very specific and help enforce a11y rules.

Reply:
They’re just going to copy & paste the tests (until we get to the “writing tests” section), so we want to have great tests. Thank you!